packages/bsdowl/bsdowl.2.2/opam

```diff
@@ -10,12 +10,15 @@ tags: [
 ]
 build: [
   ["./configure" "--prefix" prefix]
-  ["bmake" "-r" "build"]
+  ["%{conf-bmake:path}%" "-r" "build"]
 ]
-depends: ["ocamlfind"]
-depexts: [
-  [["debian"] ["bmake"]]
-  [["ubuntu"] ["bmake"]]
-  [["osx" "macports"] ["bmake"]]
-  [["freebsd"] ["devel/bmake"]]
+install: [
+  ["%{conf-bmake:path}%" "install"]
+]
+remove: [
+  ["sh" "%{build}%/opam/files/remove.sh" prefix]
```

Comment:
If this fails, the shell return an error exit code, so the `-e` is not needed here.